src/opensles/AudioStreamBuffered.h

```diff
@@ -55,6 +55,10 @@ class AudioStreamBuffered : public AudioStream {
 
     bool isXRunCountSupported() const override;
 
+    bool isMMapUsed() override {
```

Comment:
You could declare this in AudioStream.h as default and then only override it in the AAudio subclass.